src/equations/equations.jl

```diff
@@ -207,7 +207,20 @@ where `x` specifies the coordinates, `t` is the current time, and `equation` is
 struct BoundaryConditionNeumann{B}
     boundary_normal_flux_function::B
 end
+"""
+    NonConservativeLocal
```

Comment:
```suggestion
    NonConservativeLocal()
```
To clarify how it can be constructed